src/library-authoring/component-info/ComponentAdvancedInfo.tsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable import/prefer-default-export */
```

Comment:
It's not directly related here, but is there a reason why we don't remove this lint rule upstream?
Personally, I also prefer named exports.

Reply:
Good point. I guess I've just been lazy, but it's time to do that. https://github.com/openedx/eslint-config/pull/164